src/marqo/version.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "2.9.0"
+__version__ = "2.10.0"
```

Comment:
Should this be changed during the release process?

Reply:
I think it would be better if we changed the marqo version during development to the feature's corresponding marqo version. 

Is there a better way to do this?

Reply:
Do we use this info in any of the tests?

Reply:
for 2.9.0 we used the version for tests, for 2.10.0 @vicilliar might be